README.md

```diff
@@ -39,7 +39,29 @@ print(dot_dict.test.my_thing)
 # hello
 
 ```
-
+**Profiler**
+```bash
```

Comment:
We should use python instead of bash here. This gives better code highlighting. My existing examples should also be updated.

Reply:
fixed.